src/plugins/crypto/README.md

```diff
@@ -109,62 +110,62 @@ Set the CMake variables `OPENSSL_INCLUDE_DIR` and `OPENSSL_LIBRARIES` to your de
 
 The crypto plugin will encrypt and decrypt values using AES-256 in CBC mode.
 
-The key derivation is still WIP.
+The GPG binary is required for handling the cryptographic keys.
 
-### Planned Features ###
+At the moment the plugins will only run on UNIX/Linux-like systems, that implement `fork ()`.
 
-- Encryption of values
-- Decryption of values
-- Key derivation by metadata (password provided within a meta-key)
-- Key derivation by using a specified key-file (like the SSH client does)
-- Key derivation by utilizing the pgp-agent
+## Examples ##
+
+To mount a backend with the gcrypt plugin variant that uses the GPG key 9CCC3B514E196C6308CCD230666260C14A525406, use:
 
-The encryption and decryption of values is a straight forward process, once the key and IV are supplied.
-The key-derivation process in a library is a bit tricky.
+	kdb mount test.ecf user/t crypto_gcrypt "crypto/key=9CCC3B514E196C6308CCD230666260C14A525406"
 
-The crypto plugin itself can hold configuration data about the order of the applied key derivation functions.
-For example, if a password is provided by a meta-key, it will be used, otherwise we look for a key file.
-If no key file has been configured, we try to trigger the PGP agent, etc.
+Now you can specify a key `user/t/a` and protect its content by using:
 
-The following example configuration illustrates this concept:
+	kdb set user/t/a
+	kdb setmeta user/t/a crypto/encrypt 1
+	kdb set user/t/a "secret"
 
-	system/elektra/crypto/config/key-derivation/#0 = meta
-	system/elektra/crypto/config/key-derivation/#1 = file
-	system/elektra/crypto/config/key-derivation/#2 = agent
-	system/elektra/crypto/config/key-file/path/#0 = ~/.elektra/id_aes
-	system/elektra/crypto/config/key-file/path/#1 = /etc/elektra/id_aes
+The value of `user/t/a` will be stored encrypted.
+But you can still access the original value using `kdb get`:
 
-Only keys marked with a certain meta-key will be considered for encryption/decryption.
+	kdb get user/t/a
```

Comment:
Sounds like `gpg-agent` not running/not working. Is `gnupg2` installed? Did you enable the gpg-agent? I think the agent does not work with gpg v. 1 any more.